rfc/004-feature-template.md

```diff
@@ -0,0 +1,54 @@
+- Feature Name: badges_node
+- Start Date: 2020-08-24
+- RFC PR: [amundsen-io/rfcs#4](https://github.com/amundsen-io/rfcs/pull/4)
+- Amundsen Issue: [amundsen-io/amundsen#0000](https://github.com/amundsen-io/amundsen/issues/0000)
+
+# Separate badges from tags
+
+## Summary
+
+Create a new node type for neo4j which would represent Badges that is separate from the Tag node.
+
+## Motivation
+
+In order to best use badges we want to add sentiment and category fields to the Badge nodes. This makes the Tag and Badge components different, as Badge requires more information now. Separating the two now will likely lead to less tech debt in the future when other features are implemented that impact the way we ingest and use badges. Additionally, this change will help clarify the difference in use and purpose between tags and badges throughtout Amundsen. This feature will also guarantee badges are not ingested without having sentiment and category.
+
+
+## Guide-level Explanation (aka Product Details)
+
+> Explain the proposal as if it was already included in Amundsen and you were teaching it to an Amundsen user. That generally means:
```

Comment:
does section need to update ? If not, we may just delete.

Reply:
I think it should contain what you asked for in the previous comment!